packages/core-ethereum/crates/core-ethereum-actions/src/environment.rs

```diff
@@ -314,16 +314,63 @@ impl ResolvedNetwork {
     }
 }
 
+#[derive(Serialize, Clone)]
+#[cfg_attr(feature = "wasm", wasm_bindgen::prelude::wasm_bindgen(getter_with_clone))]
+pub struct DeploymentExtract {
+    pub hopr_announcements_address: String,
+    pub hopr_token_address: String,
+    pub hopr_channels_address: String,
+    pub hopr_network_registry_address: String,
+    pub hopr_node_safe_registry_address: String,
+    pub hopr_ticket_price_oracle_address: String,
```

Comment:
Please make these all of type `Address`

Reply:
Cannot do without migrating a ton of core-ethereum interfaces as well.